community-plugins.json

```diff
@@ -11445,10 +11445,17 @@
         "repo": "AndyReifman/LinkMagic"
     },
     {
-      "id": "mapbox-location",
-      "name": "Mapbox Location Image",
-      "author": "Aaron Czichon",
-      "description": "Renders an mapbox location image based on given coordinates.",
-      "repo": "aaronczichon/obsidian-location-plugin"
+        "id": "mapbox-location",
+        "name": "Mapbox Location Image",
+        "author": "Aaron Czichon",
+        "description": "Renders an mapbox location image based on given coordinates.",
+        "repo": "aa}ronczichon/obsidian-location-plugin"
```

Comment:
@joethei I think there was a merge error here in this PR :) 
Previously it was `"repo": "aaronczichon/obsidian-location-plugin"` and here it is `"repo": "aa}ronczichon/obsidian-location-plugin"`

Reply:
Fixed it here #3429